### PR TITLE
Fix/examples mariogame

### PR DESCRIPTION
**Pull request by tawfiqkhalilieh**
Fixed some bugs in Mario game like showing the pyscript code
and the 404 status code while calling the defer and pyscript stylesheets

### CODE REVIEW

File: examples/panel_stream.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
 	  <a class="navbar-brand app-logo" href="/">
 	    <img src="./logo.png" class="app-logo">
 	  </a>
-	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>
+	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming</a>
```
Comment (xtawfeeq7k) [inline examples/panel_stream.html:55]:
```suggestion
	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>**
```

**Review by xtawfeeq7k — COMMENTED** [review]
Simple changes.

File: examples/mario/play_mario.html [hunk 2]
```diff
@@ -4,15 +4,15 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
 
-    <title>Svelte app</title>
+    <title>Mrio Game - pyscript</title>
```
Comment (fpliger) [inline examples/mario/play_mario.html:7]: typo: Mrio/Mario

File: examples/mario/play_mario.html [hunk 3]
```diff
@@ -38,9 +38,7 @@
     <script type="text/javascript" src="js/levels/11tunnel.js"></script>
     <script type="text/javascript" src="js/game.js"></script>
 
-    <py-script>
-from js import handTrack, setTimeout, requestAnimationFrame, player
-from pyodide import create_once_callable
+    <py-script style="display: none;">
```
Comment (fpliger) [inline examples/mario/play_mario.html:41]: Why this change?

File: examples/mario/play_mario.html [hunk 4]
```diff
@@ -4,15 +4,15 @@
...
     <link rel="icon" type="image/png" href="../favicon.png" />
-    <link rel="stylesheet" href="../https://pyscript.net/alpha/pyscript.css" />
+    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
 
-    <script defer src="../https://pyscript.net/alpha/pyscript.js"></script>
+    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
   </head>
 
-  <body>
+  <body allow="autoplay; fullscreen">
```
Comment (fpliger) [inline examples/mario/play_mario.html:15]: why this change?

File: examples/mario/play_mario.html [hunk 5]
```diff
@@ -134,7 +132,7 @@
       <button  id="trackbutton" class="bx--btn bx--btn--secondary" type="button" pys-onClick="toggle_video">
         Start Video
       </button>
-      <div id="update-note" py-mount class="updatenote mt10">loading model ..</div>
+      <div id="update-note" py-mount class="updatenote mt10" style="display: none;"">loading model ..</div>
```
Comment (fpliger) [inline examples/mario/play_mario.html:135]: I think this change is correct. What's the intent here?

File: examples/panel_stream.html [hunk 6]
```diff
@@ -52,7 +52,7 @@
 	  <a class="navbar-brand app-logo" href="/">
 	    <img src="./logo.png" class="app-logo">
 	  </a>
-	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>
+	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>**
```
Comment (fpliger) [inline examples/panel_stream.html:55]: Why this change?

**Review by fpliger — CHANGES_REQUESTED** [review]
Thanks for the interest in PyScript and for contributing back! Unfortunately, the PR doesn't run for me locally (has errors on the console loading) and I left some comments on the PR. Can you check them when you have some time? Thank you!

File: examples/mario/play_mario.html (hunk 3 above)
Comment (xtawfeeq7k) [reply examples/mario/play_mario.html:41]: Because sometimes the DOM renders the `pyscript`'s code as a part of the HTML text

File: examples/mario/play_mario.html (hunk 2 above)
Comment (xtawfeeq7k) [reply examples/mario/play_mario.html:7]:
```suggestion
    <title>Mario Game - pyscript</title>
```

Comment (xtawfeeq7k) [reply examples/mario/play_mario.html:7]: typo: Mrio/Mario

File: examples/mario/play_mario.html (hunk 4 above)
Comment (xtawfeeq7k) [reply examples/mario/play_mario.html:15]: here I allowed autoplay for the people who accept this option from the browser settings

File: examples/mario/play_mario.html (hunk 5 above)
Comment (xtawfeeq7k) [reply examples/mario/play_mario.html:135]: Because when I was testing I saw that sometimes the loader crashes so no need to see this message if the loader is not loading, I will work on it later but for now hiding it will be the fast solution

File: examples/panel_stream.html (hunk 6 above)
Comment (xtawfeeq7k) [reply examples/panel_stream.html:55]: oh I think I added this by the mistake
```suggestion
	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>
```

File: examples/mario/play_mario.html (hunk 5 above)
Comment (xtawfeeq7k) [reply examples/mario/play_mario.html:135]: I will work on the loading errors when I have some time

**Review by xtawfeeq7k — COMMENTED** [review]
Here I fixed the URLs to make the app run-able without cloning the whole pyscript repo
The CSS Padding in this case doesn't matter for the most of the screen sizes

**Comment by marimeireles** [comment]
Due to many changes in the code base this PR is not working anymore.
@tawfiqkhalilieh really sorry for that, we've been overlooking changes in examples for a bit because we're trying to move some examples to the [pyscript-collective](https://github.com/pyscript/pyscript-collective/).
I just gave a try on rebasing it but some things are broken, maybe you're interested in checking this out again or maybe you'd prefer to wait for this to be moved to the collective.
Sorry one more time, thanks for the time here. Cheers.

**Comment by tedpatrick** [comment]
Contributed examples should be submitted as PRs to https://github.com/pyscript/pyscript-collective. We are focusing the examples within [PyScript](https://github.com/pyscript/pyscript) on feature use cases and testing.

Thanks for your patience, understanding, and contribution.
